### ISSUE TITLE
[A11Y] Unreachable labels

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1079]
## Finding
The links to the terms of service and privacy policy can not be reached by keyboard.

## Location
login.eduid.nl/request/*

``` <span>I agree with <a tabindex="-1" href="https://eduid.nl/terms-of-use/" target="_blank">the terms of service</a> and I've read <a tabindex="-1" href="https://eduid.nl/privacy" target="_blank">the privacy policy</a>.</span> ```

<img width="353" height="64" alt="" src="https://github.com/user-attachments/assets/076d2b51-c106-4371-b519-38ecc7c6f372" />

## Problem

A link that can't be reached with a keyboard, can't be used by people using all kinds of tools including keyboards.

## Solution

`tabindex="-1" ` makes an element unfocusable with a keyboard. This attribute should be removed. That leaves the issue that HTML and browsers don't like nested interactive elements. 
Ideally, move the links outside the label, and use `aria-describedby` on the checkbox the create a relationship between them.

## Resources
- https://webaim.org/techniques/keyboard/tabindex#zero-negative-one
- https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html

**Comment by Tyskai** [comment]
What is a more common design, @erikkroes ?

For example, at bol.com, I see:

<img width="1260" height="524" alt="Image" src="https://github.com/user-attachments/assets/e3d4b7e1-94c9-4125-9d20-5e0e7a4584f1" />

At HEMA:

<img width="1132" height="936" alt="Image" src="https://github.com/user-attachments/assets/16c3decd-81f9-4a7e-983a-f35ee2bf40c9" />
